From a56cf424769f28d62b465be0c562d2b581b78b3c Mon Sep 17 00:00:00 2001
From: Duy Bao Dang <duy.dang.yb@renesas.com>
Date: Fri, 7 Aug 2020 18:44:02 +0700
Subject: [PATCH 498/504] ARM: dts: r8a77470: Use SYSC power-domain for
 watchdog

RZ/G1C cannot reboot when using CPG power-domain for watchdog (rwdt).
This patch changes the power-domain to SYSC R8A77470_PD_ALWAYS_ON to
fix this issue.

Signed-off-by: Duy Bao Dang <duy.dang.yb@renesas.com>
Signed-off-by: Hao Bui <hao.bui.yg@renesas.com>
---
 arch/arm/boot/dts/r8a77470.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/r8a77470.dtsi b/arch/arm/boot/dts/r8a77470.dtsi
index 843a4ec..39ab4b1 100644
--- a/arch/arm/boot/dts/r8a77470.dtsi
+++ b/arch/arm/boot/dts/r8a77470.dtsi
@@ -107,7 +107,7 @@
 				     "renesas,rcar-gen2-wdt";
 			reg = <0 0xe6020000 0 0x0c>;
 			clocks = <&mstp4_clks R8A77470_CLK_RWDT>;
-			power-domains = <&cpg_clocks>;
+			power-domains = <&sysc R8A77470_PD_ALWAYS_ON>;
 			status = "disabled";
 		};
 
-- 
2.7.4

